sdk/storage/blobs/samples/src/blobs_client_example.c

```diff
@@ -130,19 +150,22 @@ int main()
     printf("Failed to create blob");
   }
 
+  printf("Downloading blob and printing it below:\n\n");
```

Comment:
@gilbertw: This is a bit problematic. Our "official" Storage library offers only upload. So, our sample should not show download/delete.
If you want to "officially" offer delete, we can as its fairly simple and easy to support.
However, download is VERY problematic and we should not have it at this time.

Reply:
I don't like this either. 
We are showing how to implement the Download but telling customers to access internal fields.
I would prefer to remove Download and Delete from the sample
Since we are always only uploading blob (put blob), the blob just overridden in Azure Storage, so we are not creating garbage in service that needs to be deleted at the end.

Reply:
I'm fine with moving it out of the sample and into the tests.
Its really hard to test upload without download.  We will need upload/download/delete for the test scenarios.  The later two can be in the tests only.